src/shared/extensions/git.ts

```diff
@@ -369,7 +369,7 @@ export class GitExtension {
 
                         return execFileAsync(api.git.path, ['cat-file', type, hash], {
                             cwd: tmpDir,
-                            maxBuffer: 1024 * 1024 * 5,
+                            maxBuffer: 1024 * 1024 * 6,
```

Comment:
Let's just set it to 50MB:
```suggestion
                            maxBuffer: 1024 * 1024 * 50,
```
[This thread](https://github.com/nodejs/node/issues/9829) mentions dynamic sizing a potential solution. Not sure if Node dynamically sizes the buffer already or if the buffer is allocated based off `maxBuffer`. If the former then `Infinity` could work although it would make it easier to run out of memory on the heap, crashing V8.